buf.yaml

```diff
@@ -1,10 +1,30 @@
+version: v1
 build:
-  roots:
-    - proto
-    - third_party
 lint:
   use:
-    - DEFAULT
```

Comment:
DEFAULT became more strict than it used to be; If we would use it `buf` complains about several linting errors